tests/datastore/sql/testsqlitedsdb.nim

```diff
@@ -7,6 +7,7 @@ import pkg/stew/byteutils
 import pkg/sqlite3_abi
 import pkg/datastore/key
 import pkg/datastore/sql/sqlitedsdb
+import pkg/datastore/sql/sqliteutils
```

Comment:
an import was added but nothing else in this test file changes. Is it needed?

Reply:
Yes, because `sqliteutils` is no loger exported by `sqlitedsdb` (one import above)